.github/workflows/update-googleapis-commit.yaml

```diff
@@ -0,0 +1,62 @@
+name: Update googleapis commit
+on:
+  schedule:
+  - cron: '* 2 * * *'
+  workflow_dispatch:
+
+
+jobs:
+  update-googleapis-commit:
+    runs-on: ubuntu-22.04
+    env:
+      # the branch into which the pull request is merged
+      base_branch: main
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup branch for pull request
```

Comment:
Same comment as above. Let's extract it to a script for now, and think about ways to make it more reusable.

Reply:
Moved command lines to a script.